tests/test_structs.py

```diff
@@ -116,6 +96,50 @@ def test_frozen_trial_eq_ne():
     assert trial != trial_other
 
 
+def test_frozen_trial_lt():
+    # type: () -> None
+
+    trial = _create_frozen_trial()
+
+    trial_other = copy.copy(trial)
+    assert not trial < trial_other
+
+    trial_other.number = trial.number + 1
+    assert trial < trial_other
+    assert not trial_other < trial
+
+    with pytest.raises(TypeError):
+        trial < 1
+
+    assert trial <= trial_other
+    assert not trial_other <= trial
+
+    with pytest.raises(TypeError):
+        trial <= 1
+
+    # A list of FrozenTrials is sortable.
+    trials = [trial_other, trial]
+    trials.sort()
+    assert trials[0] == trial
```

Comment:
A nitpick but you can probably check for object identity here.
```suggestion
    assert trials[0] is trial
```

Reply:
Thank you for the careful review. You're absolutely right.
Fixed: https://github.com/optuna/optuna/pull/719/commits/0245b32945a696acbe28da7e282070ef07f555b5